mapdata/WarcraftLegacies/AbilityData/Skin/843460673.json

```diff
@@ -0,0 +1,23 @@
+{
+  "OldId": 1684095809,
+  "NewId": 843460673,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(Twilight DK)"
```

Comment:
The data references in this ability tooltip are wrong. They currently refer to the ability `ACad`, which is not this one. Also Twilight isn't in the game, so is there any point adding this?

Reply:
Was just ability i copy and pasted for the new DK one. Will remove